lint.js

```
@@ -1,3 +1,13 @@
 const add = function(a, b) {
     return a + b;
 }
+const add2 = function(a, b) {
+    return a + b;
+}
+const equal = function(a, b) {
+    return a == b;
+}
+const request = async function() {
```

ESLint

[error] 'request' is assigned a value but never used.

(no-unused-vars)